portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/SPA.testcase

```diff
@@ -33,4 +33,86 @@ definition {
 		AssertVisible.assertVisibleNoSPAWait(locator1 = "//html[contains(@class,'lfr-spa-loading')]");
 	}
 
+	@description = "Validate user can configure SPA timeout notification"
+	@priority = "4"
+	test ConfigureSPANotificationTimeout {
+		property test.name.skip.portal.instance = "SPA#AssertLoadingBar";
+
+		var appName = "Liferay Frontend JS SPA Web";
+
+		task ("Configure User Notification Timeout to 1 second") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "System Settings");
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "Infrastructure",
+				configurationName = "Frontend SPA Infrastructure",
+				configurationScope = "System Scope");
+
+			Type(
+				key_text = "User Notification Timeout",
+				locator1 = "TextInput#ANY",
+				value1 = "1000");
+
+			SystemSettings.saveConfiguration();
+		}
+
+		task ("Reset the SPA module in App Manager to fully apply the SPA configuration") {
```

Comment:
I'm not sure why this is needed. Can't you just do a manual refresh? I think that's the only thing that's actually making the setting take effect again.

Reply:
huh weird, I thought I tried the refresh route before and it didn't work. But now it works...ok will change this.